sshcommand

```diff
@@ -58,6 +58,12 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=$NAME" "$USERHOME/.ssh/authorized_keys" || true)
```

Comment:
I don't think this works with the new quoting stuff. Would you mind adding a test to check for duplicate keys?